includes/post-hooks.php

```
@@ -569,6 +518,37 @@ function push_story_update( int $post_id, string $story_id, array $update_data,
+		);
+		return;
+	}
+
+	update_story_state(
+		$post_id,
+		array(
+			'status'  => StoryStatus::Error->value,
+			'message' => $result['message'],
+		)
+	);
+}
```

PHPMD (2.15.0)

[error] The parameter $post_id is not named in camelCase. (undefined)

(CamelCaseParameterName)